test/integration/resource-manager.v2.test.js

```diff
@@ -30,25 +31,47 @@ const describe = authHelper.prepareTests(configFile);
 describe('ResourceManagerV2_integration', () => {
   jest.setTimeout(timeout);
 
-  let service1;
-  let service2;
-  let new_resource_group_id;
-  const test_quota_id = '7ce89f4a-4381-4600-b814-3cd9a4f4bdf4';
-  const test_user_account_id = '60ce10d1d94749bf8dceff12065db1b0';
-
-  it('should successfully complete initialization', done => {
-    service1 = ResourceManagerV2.newInstance({ serviceName: 'RMGR1' });
-    expect(service1).not.toBeNull();
-
-    service2 = ResourceManagerV2.newInstance({ serviceName: 'RMGR2' });
-    expect(service2).not.toBeNull();
+  let resourceManagerService = null;
+  let resourceManagerUsersService = null;
+  let newResourceGroupId = null;
+  let url = null;
+  let authType = null;
+  let apiKey = null;
+  let authUrl = null;
+  let testQuotaId = null;
+  let testUserAccountId = null;
+
+  beforeAll('should successfully complete initialization', done => {
```

Comment:
The first argument is a callback function, not a string. Check the other integration tests (search for `beforeAll` in the project)